chapter07/pxd/ex1.pl

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use feature "say";
+
+say "Enter a couple of lines (Ctrl-D to exit):";
+
+while (<>) {
+    
+    # Will match 'freddie'
+    if (/fred/) { 
+    	say "The entered line contains the letters fred "; 
+    }
+    
+    # will match only 'fred' 
+    if ( /(^|\s)fred(\s|$)/ ) {
```

Comment:
This is an interesting way of handling word boundaries without cheating and including something from the next chapter :)